src/connection.rs

```diff
@@ -705,38 +705,31 @@ impl Connection {
             return;
         }
 
-        self.update_client_quality_score();
+        self.refresh_client_conn_quality_score();
     }
 
-    /// Updates [`ClientConnectionQualityScore`] of this [`Connection`].
-    fn update_client_quality_score(&self) {
+    /// Refreshes [`ClientConnectionQualityScore`] of this [`Connection`].
+    fn refresh_client_conn_quality_score(&self) {
+        use PeerConnectionState as S;
+
         let state = self.0.peer_state.get();
         let quality_score = self.0.quality_score.get();
-
         let score = match (state, quality_score) {
-            (
-                Some(
-                    PeerConnectionState::Disconnected
-                    | PeerConnectionState::Failed,
```

Comment:
Simply add a local `use FooBar as F` import to make these matches more readable.